Packages/vcs/Lib/VTKPlots.py

```diff
@@ -87,7 +90,7 @@ def endEvent(self,obj,event):
     if self.renWin is not None:
       if self.reRender:
         self.reRender = False
-        self._lastSize = None
+        #self._lastSize = None
```

Comment:
Have you tried resizing the UVCDAT window while replaying an animation? It's been a while, but I believe that was the issue.